### PR TITLE
Add note about bower and gulp dependency

### PR DESCRIPTION
**Pull request by ThomasWo**
Installing from a clean slate, `polymer init <blah>` fails silently without bower (and gulp is needed in there somewhere).  Thus I suggest adding a note about requiring the use of bower and gulp.

### CODE REVIEW

**Comment by ThomasWo** [comment]
I signed it!

File: app/1.0/start/toolbox/set-up.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ App Toolbox template in less than 15 minutes.
 1.  Install the LTS version (4.x) of Node.js. The current version (6.x) should work, but is not
     officially supported. Versions below LTS are not supported.
 
+1. If you don't have them installed, install bower and gulp
+
+        npm install -g bower gulp
```
Comment (addyosmani) [inline app/1.0/start/toolbox/set-up.md]: @justinfagnani Do we currently make an attempt to resolve/install these dependencies or warn the user about them not being available or is this line worth adding to the setup docs for toolbox?

Comment (FredKSchott) [inline app/1.0/start/toolbox/set-up.md]: Gulp is no longer required by polymer-cli (it used to be). No need to mention here anymore.

**Comment by ThomasWo** [comment]
@FredKSchott I've removed the gulp reference in the file, committed, and squashed.  PTAL

File: app/1.0/start/toolbox/set-up.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ App Toolbox template in less than 15 minutes.
 1.  Install the LTS version (4.x) of Node.js. The current version (6.x) should work, but is not
     officially supported. Versions below LTS are not supported.
 
+1. If you don't have them installed, install bower and gulp
```
Comment (justinfagnani) [inline app/1.0/start/toolbox/set-up.md]: If you don't have bower installed, install it:

**Comment by justinfagnani** [comment]
@ThomasWo are you using two different email addresses for the CLA and for commits?

**Comment by ThomasWo** [comment]
@justinfagnani Looks like the git config on one of my machines was set to the wrong account for some reason, so yes, but it shouldn't be a problem as both accounts have signed the CLA.

**Comment by justinfagnani** [comment]
Thanks @ThomasWo!

**Pull request merged**